prereise/gather/data/remap_ba_area/ercot_zone_mapping.csv

```diff
@@ -0,0 +1,196 @@
+﻿County,WeatherZone,Zone

```

Comment:
Is the information in the `WeatherZone` and `Zone` column redundant?

Reply:
the WeatherZone is useless, and dropped from the csv file.